docs/python/python-tutorial.md

```diff
@@ -252,36 +253,33 @@ A best practice among Python developers is to avoid installing packages into a g
 
 > **Note**: For additional information about virtual environments, see [Environments](/docs/python/environments.md#creating-environments).
 
-1. Create and activate the virtual environment
+1. Create a Virtual Environment using the Create Environment Command
 
-   **Virtual environment creation for Windows**
+   From within VS Code, you can create non-global environments, using Venv or Anaconda, by opening the Command Palette (Ctrl+Shift+P), start typing the `Python: Create Environment` command to search, and then select the command. You can also trigger the `Python: Create Environment' command through the Getting Started with Python page.
```

Comment:
You can use ``kb(workbench.action.showCommands)`` instead of directly putting the Windows/Linux keyboard shortcuts. The website will then show the correct keyboard shortcuts depending on the reader's platform.

Reply:
There is an example above in the Select a Python interpreter section

Reply:
Also please use bold for command text so **Python: Create Environment**

Reply:
Fixed